experiment/simulation/index.html

```diff
@@ -1,13 +1,339 @@
+                    />
+                  </g>
+                  <g fill="none" id="pink-bottom">
+                    <path
+                      d="M169 280.5 c0 -0.8 0.5 -1.5 1.1 -1.5 0.6 0 2.2 -1.3 3.6 -3 1.3 -1.6 3 -3 3.6 -3 0.7 0 2.6 -0.7 4.2 -1.6 2 -1.1 7.4 -1.9 16.5 -2.5 7.4 -0.5 13.6 -1.3 13.8 -1.8 0.2 -0.6 3 -1.3 6.3 -1.7 3.2 -0.4 5.9 -1.1 5.9 -1.5 0 -1.3 20.7 -1.1 21.5 0.1 0.3 0.6 3.8 1 7.6 1 13.3 0 23.3 4.2 24.2 10.1 0.2 1.3 1 2.5 1.8 2.7 1.1 0.3 1.2 0.2 0.1 -0.6 -1.1 -0.8 -1.1 -1 0.1 -1.5 0.8 -0.3 2.4 0.3 3.5 1.4 1.2 1 3 1.9 4.2 1.9 1.1 0 2 0.7 2 1.5 0 1.3 -7.3 1.5 -60 1.5 -52.7 0 -60 -0.2 -60 -1.5z"
+                    />
+                  </g>
+  
+                  <g fill="#000000">
+                    <path
+                      d="M139 186.9 c0 -80.4 0.2 -95 1.4 -94.6 0.7 0.3 1.7 0.2 2.2 -0.3 0.5 -0.4 5.8 -2.7 11.7 -5 19.8 -7.7 29.3 -15.9 40.6 -34.8 l5.1 -8.6 -2.5 -2.1 c-2.5 -1.9 -2.6 -2.4 -2.3 -10.5 l0.3 -8.5 4 -2 c3.3 -1.7 6 -2 15.5 -2 9.5 0 12.2 0.3 15.5 2 l4 2 0 9 0 9 -4 2 c-3.3 1.6 -6.1 2 -15.8 2.1 l-11.8 0.1 -5.2 8.8 c-11.1 18.8 -21.9 28.2 -41.3 36 -9.3 3.7 -10.4 4.4 -8.7 5.3 4.8 2.6 38.7 13.3 48.3 15.3 14.7 3.1 33.5 3.1 48.9 0 10.4 -2 39.6 -10.9 48.8 -14.8 l3.8 -1.6 -11 -2.9 c-16 -4.2 -27.9 -10.3 -37.1 -19 -7.5 -7.1 -21.4 -24.4 -21.4 -26.6 0 -2.7 2.2 -0.8 8.2 7.1 17 22.4 27.5 29.5 54.8 36.6 l10.5 2.8 0.3 92.9 c0.1 51.1 0 93.9 -0.3 95.2 -0.5 2 -1.1 2.2 -7.6 2.2 -4.9 0 -7.5 -0.5 -8.5 -1.5 -1.4 -1.3 -0.7 -1.5 6 -1.5 l7.6 0 0 -91 c0 -50 -0.2 -91 -0.3 -91 -0.2 0 -6.9 2.3 -14.8 5.1 -28.4 10 -53.1 14.8 -69.8 13.4 -16.9 -1.4 -36.5 -6.3 -59.6 -14.8 -6.6 -2.5 -12.1 -4.4 -12.2 -4.3 -0.4 0.4 -0.4 181.9 0 182.2 0.2 0.2 31.6 0.4 69.9 0.4 60.5 0 69.5 0.2 69.5 1.5 0 1.3 -9.2 1.5 -71.4 1.5 l-71.3 0 0 -95.1z m90.8 -147.3 c3.1 -1.6 3.2 -1.8 3.2 -8.1 0 -6.2 -0.1 -6.5 -3.2 -8.2 -4.7 -2.6 -24.9 -2.6 -29.5 0 -3.2 1.7 -3.3 2 -3.3 8.2 0 7 0.7 7.8 7.5 9.6 5.2 1.4 21.6 0.4 25.3 -1.5z"
+                    />
+                  </g>
+                  <text x="25%" y="82%" class="heavy">Solution Beaker</text>
+                </svg>
+              </div>
+              
+            
+            <div class="apparatus-desiccator column" id="apparatus-desiccator">
+              <div class="row desiccator-row" id="desiccator-row">
+                <figure>
+                  <img
+                    src="./images/desiccator.png"
+                    alt="This is an animated gif image"
```

Comment:
Please change the alt attribute to describe the image meaningfully. Web crawlers use the alt attribute to understand an image.